packages/main/types/requests.ts

```diff
@@ -147,6 +147,11 @@ export interface RequestOptions {
    * Custom HTTP request headers.
    */
   customHeaders?: Headers | Record<string, string>;
+  /**
+   * An object that may be used to abort aynchronous requests. This is only available on
```

Comment:
If we separate RequestOptions into two types, we don't need the "and is ignored by models..." part, although the "only available" part might be helpful. We should also add how this works together with timeout (whichever one triggers first). Also spelling on "asynchronous"

Reply:
Done.